assets/js/base/components/price-slider/index.js

```diff
@@ -133,11 +141,11 @@ const PriceSlider = ( {
 			 * slider should be at the front and has no meaning beyond
 			 */
 			if ( minXDiff > maxXDiff ) {
-				minRange.current.style.zIndex = 20;
-				maxRange.current.style.zIndex = 21;
+				currentMinRange.style.zIndex = 20;
+				currentMaxRange.style.zIndex = 21;
 			} else {
-				minRange.current.style.zIndex = 21;
-				maxRange.current.style.zIndex = 20;
+				currentMinRange.style.zIndex = 21;
+				currentMaxRange.style.zIndex = 20;
```

Comment:
There's an assumption here that `style` exists as a property on the range values. There's risk here of getting assignment warnings/error for `zIndex`.